Challenge_3.rb

```diff
@@ -0,0 +1,48 @@
+Bank Transactions
+
+#display their balance, withdraw or deposit
+
+
+puts "What do you want to do?"
+balance = 4000
+input = gets.chomp
+
+def display_balance(param)
+  puts "$4000"
+  num1 = gets.chomp.to_i
+  puts "What do you want to do?"
+end
+
+def withdraw
+print 'how much?'
+withdraw_amount = gets.chomp.to_i
+balance - withdrawl_amount = balance
+print 'your balance is now #{balance}'
+end
+
+def deposit(balance, deposit_amount)
+
+balance + deposit_amount = balance
+end
+
+
+if input == 'display balance'
+  balance
+
+
+elsif input == 'deposit'
+print 'how much?'
+deposit_amount = gets.chomp.to_i
+deposit(balance, deposit_amount)
+
+
+elsif input == 'withdraw'
+print 'how much?'
+withdraw_amount = gets.chomp.to_i
+withdraw(balance, withdraw_amount)
+
+ withdraw = gets.chomp.to_i
+
+
+end
+
```

Comment:
Here is an alternative example:

balance = 4000

 puts "Your current balance is: '#{balance}'"

 puts  'What would you like to do? (deposit, withdraw, check_balance)'
 calculation = gets.chomp.strip

     if calculation == 'deposit'
       puts "How much would you like to deposit?"
       num1 = gets.chomp.strip
       balance = num1.to_i + balance.to_i
     elsif calculation == 'withdraw'
       puts "How much would you like to withdraw?"
       num1 = gets.chomp.strip
       balance = balance.to_i - num1.to_i
     elsif calculation == 'check_balance'
       puts balance
     else
 end


puts "Your current balance is '#{balance}'"